### PR TITLE
feat: Add blur function to ISearchBox componentRef

### LINKED ISSUE #27238: [Feature]: expose blur() for ISearchBox's ref.
### Library

React / v8 (@fluentui/react)

### Describe the feature that you would like added

Hello, 
We at Viva Engage are using this SearchBox to support an autocomplete-enabled search input, and need a clean way to blur the focus out of the input upon clicking an autocompleted item. Just found out that while focus() and hasFocus() are exposed, we do not have a way to use blur().

Can we have that opened in the ref implementation? 

Thank you. 

### Have you discussed this feature with our team

_No response_

### Additional context

_No response_

### Validations

- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### PR DESCRIPTION
**Pull request by behowell**
## Previous Behavior

The ISearchBox component ref interface does not have a blur function.

## New Behavior

Add a `blur()` function to ISearchBox interface, which blurs the SearchBox's input element.

## Related Issue(s)

- Fixes #27238

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |         Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              | `67.994 kB`<br />`18.395 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> |  `207.4 kB`<br />`57.516 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            | `38.819 kB`<br />`12.351 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |   `6.446 kB`<br />`2.185 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/4fad4cbbcbdb48847857c388135a3abda472add7'>4fad4cbbcbdb48847857c388135a3abda472add7</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test-react-components/Avatar-mount_base.html">874</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test-react-components/Avatar-mount.html">902</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test-react-components/Button-mount_base.html">598</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test-react-components/Button-mount.html">582</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test-react-components/Field-mount_base.html">1552</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test-react-components/Field-mount.html">1553</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test-react-components/FluentProvider-mount_base.html">1109</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test-react-components/FluentProvider-mount.html">1096</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test-react-components/FluentProviderWithTheme-mount_base.html">316</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test-react-components/FluentProviderWithTheme-mount.html">313</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">291</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">289</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">303</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">303</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test-react-components/InfoButton-mount_base.html">204</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test-react-components/InfoButton-mount.html">207</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test-react-components/MakeStyles-mount_base.html">1361</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test-react-components/MakeStyles-mount.html">1359</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test-react-components/Persona-mount_base.html">2149</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test-react-components/Persona-mount.html">2050</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test-react-components/SpinButton-mount_base.html">1846</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test-react-components/SpinButton-mount.html">1817</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/BaseButton-mount_base.html">944</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/BaseButton-mount.html">943</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Breadcrumb-mount_base.html">2184</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Breadcrumb-mount.html">2168</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Checkbox-mount_base.html">2165</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Checkbox-mount.html">2144</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/CheckboxBase-mount_base.html">1952</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/CheckboxBase-mount.html">1931</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/ChoiceGroup-mount_base.html">3645</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/ChoiceGroup-mount.html">3661</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/ComboBox-mount_base.html">920</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/ComboBox-mount.html">942</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/CommandBar-mount_base.html">7848</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/CommandBar-mount.html">7833</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/ContextualMenu-mount_base.html">14308</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/ContextualMenu-mount.html">13664</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/DefaultButton-mount_base.html">1085</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/DefaultButton-mount.html">1074</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/DetailsRow-mount_base.html">2791</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/DetailsRow-mount.html">2777</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/DetailsRowFast-mount_base.html">2799</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/DetailsRowFast-mount.html">2774</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/DetailsRowNoStyles-mount_base.html">2597</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/DetailsRowNoStyles-mount.html">2608</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Dialog-mount_base.html">3156</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Dialog-mount.html">3153</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/DocumentCardTitle-mount_base.html">412</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/DocumentCardTitle-mount.html">417</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Dropdown-mount_base.html">2604</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Dropdown-mount.html">2626</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/FocusTrapZone-mount_base.html">1550</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/FocusTrapZone-mount.html">1560</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/FocusZone-mount_base.html">1502</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/FocusZone-mount.html">1513</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/GroupedList-mount_base.html">45800</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/GroupedList-mount.html">53350</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/GroupedList-virtual-rerender_base.html">22502</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/GroupedList-virtual-rerender.html">22443</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">63720</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/GroupedList-virtual-rerender-with-unmount.html">63627</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/GroupedListV2-mount_base.html">414</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/GroupedListV2-mount.html">410</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/GroupedListV2-virtual-rerender_base.html">391</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/GroupedListV2-virtual-rerender.html">389</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">407</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">422</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/IconButton-mount_base.html">1460</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/IconButton-mount.html">1459</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Label-mount_base.html">554</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Label-mount.html">563</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Layer-mount_base.html">3492</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Layer-mount.html">3514</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Link-mount_base.html">648</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Link-mount.html">651</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/MenuButton-mount_base.html">1312</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/MenuButton-mount.html">1310</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/MessageBar-mount_base.html">25783</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/MessageBar-mount.html">26033</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Nav-mount_base.html">2487</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Nav-mount.html">2471</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/OverflowSet-mount_base.html">1124</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/OverflowSet-mount.html">1128</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Panel-mount_base.html">2020</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Panel-mount.html">2067</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Persona-mount_base.html">1027</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Persona-mount.html">1060</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Pivot-mount_base.html">1192</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Pivot-mount.html">1216</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/PrimaryButton-mount_base.html">1201</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/PrimaryButton-mount.html">1187</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Rating-mount_base.html">5838</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Rating-mount.html">5820</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/SearchBox-mount_base.html">1216</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/SearchBox-mount.html">1211</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Shimmer-mount_base.html">2441</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Shimmer-mount.html">2466</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Slider-mount_base.html">1739</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Slider-mount.html">1748</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/SpinButton-mount_base.html">3557</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/SpinButton-mount.html">3783</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Spinner-mount_base.html">608</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Spinner-mount.html">652</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/SplitButton-mount_base.html">2382</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/SplitButton-mount.html">2329</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Stack-mount_base.html">663</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Stack-mount.html">657</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/StackWithIntrinsicChildren-mount_base.html">1212</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/StackWithIntrinsicChildren-mount.html">1222</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/StackWithTextChildren-mount_base.html">3510</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/StackWithTextChildren-mount.html">3519</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/SwatchColorPicker-mount_base.html">7521</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/SwatchColorPicker-mount.html">7559</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/TagPicker-mount_base.html">1860</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/TagPicker-mount.html">1871</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Text-mount_base.html">616</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Text-mount.html">627</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/TextField-mount_base.html">1267</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/TextField-mount.html">1241</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/ThemeProvider-mount_base.html">1102</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/ThemeProvider-mount.html">1128</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/ThemeProvider-virtual-rerender_base.html">902</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/ThemeProvider-virtual-rerender.html">912</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1645</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1648</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Toggle-mount_base.html">895</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/Toggle-mount.html">893</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/buttonNative-mount_base.html">376</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27330/perf-test/buttonNative-mount.html">369</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: packages/react/src/components/SearchBox/SearchBox.types.ts [hunk 1]
```diff
@@ -13,6 +13,11 @@ export interface ISearchBox {
    */
   focus(): void;
 
+  /**
+   * Blur focus from the search input box.
```
Comment (khmakoto) [inline packages/react/src/components/SearchBox/SearchBox.types.ts:17]:
```suggestion
   * Blurs focus from the search input box.
```

**Review by khmakoto — APPROVED** [review]

File: change/@fluentui-react-a322dc84-1336-4f83-af8e-fea2ad52f47b.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-a322dc84-1336-4f83-af8e-fea2ad52f47b.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27330&commitId=4025ee7cb478f147bdf38ce531e5c4515cf7ea3c&env=prod)

✅ There was 2 screenshots added, 0 screenshots removed, 1045 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv8"/>

**Pull request merged**